test/integration/conftest.py

```diff
@@ -66,8 +71,14 @@ def create_context(
     return create_context
 
 
+ApiContextProvider = Callable[
```

Comment:
This doesn't work too well, you need to use NewType, because otherwise the type checker can't work with that

Reply:
Fixed